README.md

```diff
@@ -86,11 +89,26 @@ You may need to run this with `sudo`, or from within an Administrator command sh
 
 -> You can use Example Thing node on Node-RED flow editor.
 
+### Example 5. Create an original node from SUBFLOW definition
+
+- In Node-RED flow editor, create the SUBFLOW template with module properties filled.  Here, we assume module name for the SUBFLOW template is node-red-contrib-qrcode.
+- Exprot JSON definition of the SUBFLOW template from Export menu.  We assume it is saved to a file named qrcode.json.
+- node-red-nodegen qrcode.json
+- cd node-red-contrib-qrcode
+- npm install
```

Comment:
Do you still need to do this linking these days ? - I thought you can just go into your .node-red dir - then `npm install {path-to}/node-red-contrib-qrcode` and it would do the necessary install and create links automatically.

Reply:
I also don't use npm link these days.
I've tried to match this part with other parts, but instead fixed  not to use npm link in the new commit.